kairon/shared/chat/agent/agent_flow.py

```
@@ -0,0 +1,287 @@
+                if val := slot_vals.get(slot.name):
+                    slot.initial_value = val
+                slot_args = {arg: getattr(slot, arg, None) for arg in slot_defination['args']}
+                slot_constructor = slot_defination['constructor']
+                slots.append(slot_constructor(**slot_args))
+
+            slots.append(TextSlot(name="agentic_flow_bot_user", initial_value=self.bot_settings.user, value_reset_delay=None, influence_conversation=False, mappings=[]))
+
+            return slots
+        except Exception as e:
+            logger.error(f"Error in loading slots: {e}")
+            raise AppException(f"Error in loading slots: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)